service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/EhrDocumentMapper.java

```diff
@@ -25,14 +25,15 @@
         .appendPattern("yyyyMMddHHmmss")
         .parseDefaulting(ChronoField.NANO_OF_DAY, 0)
         .toFormatter();
+    private static final String UK_ZONE_ID = "Europe/London";
 
     private final TimestampService timestampService;
     private final RandomIdGeneratorService randomIdGeneratorService;
 
     public EhrDocumentTemplateParameters mapToMhsPayloadTemplateParameters(GetGpcDocumentTaskDefinition taskDefinition, String messageId) {
         return EhrDocumentTemplateParameters.builder()
             .resourceCreated(DATE_TIME_FORMATTER.format(timestampService.now()
-                .atOffset(ZoneOffset.UTC)))
+                .atZone(ZoneId.of(UK_ZONE_ID))))
```

Comment:
Use DateFormatUtil instead?

Reply:
It does not support "Instant" types. Shall I convert them to Date time?

Reply:
Add a method for producing the string from instant